lib/nightmare.js

```diff
@@ -80,8 +80,9 @@ function Nightmare(options) {
 
   process.setMaxListeners(Infinity);
   process.on('uncaughtException', function(err) {
-    console.error(err.stack);
+    debug(err.stack);
```

Comment:
I switched to `debug` just because re-throwing the error will bubble through to a node handler which logs the stack trace to `stderr`. Adding `console.error` here means the stack gets printed twice.

The down side to re-throwing is it shows the error originating from here, however rest of the error stack is normal.